applications/filestore-backup/values.yaml

```diff
@@ -0,0 +1,67 @@
+# Default values for filestore-backup.
+
+# -- Override the base name for resources
+nameOverride: ""
+
+# -- Override the full name for resources (includes the release name)
+fullnameOverride: ""
+
+# -- Resource limits and requests for the filestore-backup pods
+resources: {}
+
+# -- Annotations for the filestore-backup pods
+podAnnotations: {}
+
+# -- Node selector rules for the filestore-backup pods
+nodeSelector: {}
+
+# -- Tolerations for the filestore-backup pods
+tolerations: []
+
+# -- Affinity rules for the filestore-backup pods
+affinity: {}
+
+# The following will be set by parameters injected by Argo CD and should not
+# be set in the individual environment values files.
+image:
+  # -- Filestore-Backup image to use
+  repository: ghcr.io/lsst-sqre/rubin-google-filestore-tool
+  # -- Pull policy for the filestore-backup image
+  pullPolicy: "IfNotPresent"
+  # -- Tag of filestore-backup image to use
+  # @default -- The appVersion of the chart
+  tag: ""
+
+tool:
+  # -- Filestore instance (e.g. "fshare-instance-dev")
+  # @default -- Must be overridden in environment-specific values file
+  instance: ""
+  # -- Zone for Filestore instance (e.g. "b" from "us-central1-b")
+  # @default -- Must be overridden in environment-specific values file
+  zone: ""
+  # -- File Share name for filestore instance.  Always "share1" unless
+  # storage is on an Enterprise tier
+  fileShare: "share1"
+  backup:
+    # -- Turn on debugging mode
+    debug: false
+    # -- Backup schedule
+    schedule: "0 10 * * *"
+  purge:
+    # -- Turn on debugging mode
+    debug: false
+    # -- purge schedule
+    schedule: "45 10 * * *"
+    # -- Number of backups to keep when purging
+    keep: 6
+
+global:
+  # -- Base path for Vault secrets
+  # @default -- Set by Argo CD
+  vaultSecretsPath: ""
+  # -- GCP Project ID
+  # @default -- Set by Argo CD
+  gcpProjectId: ""
+  # -- GCP Region
+  # @default -- Set by Argo CD
+  gcpRegion: ""
```

Comment:
Injecting these is a good idea since we know them already, but if you add any new injections, you need to also update `phalanx.services.application.ApplicationService._build_injected_values` to add the additional values.